src/runway/commands/modules_command.py

```diff
@@ -525,14 +520,11 @@ def _deploy_module(self, module, deployment, context):
             module_opts['environments'] = deployment['environments'].copy()  # noqa
         if deployment.get('module_options'):
             module_opts['options'] = deployment['module_options'].copy()  # noqa
-        if isinstance(module, six.string_types):
-            module = {'path': module}
-        if path_is_current_dir(module['path']):
-            module_root = self.env_root
-        else:
-            module_root = os.path.join(self.env_root, module['path'])
+
+        path = Path(module, self.env_root, os.path.join(self.env_root, '.runway_cache'))
```

Comment:
Would it be better to initialize this class when the config file is being parsed?

Reply:
I didn't want to mess around too much in modules_command initially just because I'm so new to the ecosystem and this is already a fairly large PR. I think this is worth looking at as a refactor effort where I could garner a little more guidance from you around what's advantageous or not.